index.js

```diff
@@ -39,16 +62,41 @@ const getAccounts = () => {
   })
 }
 
+const getSpecialTemplateParams = (command) => {
+  let returnType = ""
+  let returnExpression = ""
+
+  const pluck = (arr, key) => arr.map(item => item[key])
+
+  returnType = pluck(specialGlobals[command], "returnType").join(", ")
+  returnExpression = pluck(specialGlobals[command], "prop")
+  returnExpression = "(" + returnExpression.map(item => command + "." + item).join(", ") + ");"
```

Comment:
Great, this is the right approach! Now, we can remove the function completely and just use the pluck + join expressions below where `getSpecialTemplateParams` is called. That eschews the extra function and keeps things cleaner since it is a small amount of code.